platform/packages/versioning/src/release.rs

```diff
@@ -9,6 +11,21 @@ use super::{Version, VersionSegment};
 #[serde(transparent)]
 pub struct ReleaseLabel(&'static str);
 
+impl Display for ReleaseLabel {
+    fn fmt(&self, f: &mut Formatter<'_>) -> FmtResult {
+        f.write_str(self.0)
+    }
+}
+
+impl<S> PartialEq<S> for ReleaseLabel
```

Comment:
not a good idea to allow `release == "some_txt"` ... in general, there should be a solid reason to justify such a case. Do we have?

Reply:
Yes, indeed we do. There is absolutely no other way to compare it with the passed string which represents the expected one, if we don't want to allow it to be built from outside it's package.

Reply:
as usually, the need of such a mild hacks come due to errors in the design. In our case, that is due to the misusage of `String` where `ReleaseLabel` should be.